branch-network-formula/branch-network/map.jinja

```diff
@@ -0,0 +1,21 @@
+{% set map = salt['grains.filter_by']({
+  'SL-Micro': {
+        'containerized': True,
+        'transactional': True,
+        'network': 'networkmanager',
+        'firewall': 'firewalld' 
+  },
+  'SLES': {
+        'containerized': False,
+        'transactional': False,
+        'network': 'netconfig',
```

Comment:
SLES-16 is using networkmanager too and can be transactional.

Grain `transactional` tells you if the system is transactional or not.